### ISSUE TITLE
Error on LocalBinaryPattern public object clone()

### ISSUE THREAD

**Issue by blaisexen** [accord-net/framework#438]
my solution was to comment 3 lines:
```
        public object Clone()
        {
            var clone = new LocalBinaryPattern(blockSize, cellSize, normalize);
            //clone.epsilon = epsilon;
            //clone.histograms = (int[,][])histograms.Clone(); //**ERROR HERE**
            //clone.patterns = (int[,])patterns.Clone(); //**ERROR HERE**
            return clone;
        }
```

**Comment by cesarsouza** [comment]
Thanks for opening the issue! Probably histograms and patterns were null during the cloning. I will add a fix in the following minutes.

**Comment by blaisexen** [comment]
Oh, it is really an issue, I'll wait for that in this lines,
thank you

**Comment by blaisexen** [comment]
but the code I used is this:
```
                var binarySplit = new BinarySplit(numberOfWords);
                var freak = new LocalBinaryPattern();
                var freakBow = new BagOfVisualWords<FeatureDescriptor>(freak, binarySplit);
```

maybe it doesn't affect

**Comment by cesarsouza** [comment]
Your fix is fine in this situation. It will work fine with BagOfVisualWords.

**Comment by blaisexen** [comment]
issue closed, and maybe fixed for next update released.
thank you

**Comment by cesarsouza** [comment]
Fixed in 3.5.0.
